app/subscriber/src/features/impersonation/Impersonation.tsx

```diff
@@ -95,7 +95,7 @@ export const Impersonation = () => {
             <Col flex="1">Username</Col>
             <Col flex="1">Email</Col>
             <Col flex="1">First Name</Col>
-            <Col flex="1">Last Name</Col>
+            <Col flex="1">Last Name</Col>Impersonate
```

Comment:
Was this intentional?

Reply:
Good catch, I have no idea how that happened... removed